impl/src/main/java/org/apache/myfaces/renderkit/html/base/HtmlTextRendererBase.java

```diff
@@ -142,8 +142,8 @@ protected void renderOutput(FacesContext facesContext, UIComponent component) th
         }
         else
         {
-            //default is to escape
-            escape = AttributeUtils.getBooleanAttribute(component, ComponentAttrs.ESCAPE_ATTR, true);
+            //default is NOT to escape (Changed true -> false, Myfaces-4477)
+            escape = AttributeUtils.getBooleanAttribute(component, ComponentAttrs.ESCAPE_ATTR, false);
```

Comment:
isnt this a bug security flaw?

Reply:
I've removed the changes and updated the test to not escape the <html> elements. 

I will bring this escaping issue up to the TCK.